docs/docs/recipes.md

```diff
@@ -853,6 +853,113 @@ export const pageQuery = graphql`
 - [Guide to creating pages from data programmatically](/docs/programmatically-create-pages-from-data/)
 - [Example repo](https://github.com/gatsbyjs/gatsby/tree/master/examples/recipe-sourcing-markdown) for this recipe
 
+### Sourcing data from Contentful
+
+#### Prerequisites
+
+- A [Contentful account](https://www.contentful.com/)
+- A [Contentful space](https://www.contentful.com/faq/basics/#how-do-i-create-a-space)
+- One [Content model](https://www.contentful.com/developers/docs/concepts/data-model/)
+- One or more pieces of published content to [avoid build errors](https://github.com/gatsbyjs/gatsby/issues/15344#issuecomment-520151776)
+- The [space ID and content delivery API key](https://www.contentful.com/developers/docs/concepts/apis/#content-delivery-api) for your space
+
+The screenshots and examples in this recipe were set up with a new Contentful space with a "News" content model that includes "Title" and "Body" fields.
```

Comment:
When going through this recipe Title & Body don't have the same type.

`Title` is a short text type & `Body` is a long text type. I'm not sure if we should make this a bit more clear.

Reply:
Agreed, added in https://github.com/kevee/gatsby/commit/95facc99e25331a5e576db366590b56f62e7cb99